addons/web/static/src/js/libs/bootstrap.js

```diff
@@ -107,6 +107,17 @@ $.fn.scrollspy.Constructor.prototype._process = function () {
     }
 };
 
+/**
+ * Prevents 'active' class removal for top menu items.
+ */
+const bootstrapSpyClearFunction = $.fn.scrollspy.Constructor.prototype._clear;
+$.fn.scrollspy.Constructor.prototype._clear = function () {
+    if (!this._config.target) {
+        this._selector = this._selector.replace('.nav-link', '.nav-link:not(#top_menu .nav-link)');
```

Comment:
With the solution you propose, we would only solve the problem for 'nav-link' in '#top_menu', but this Bootstrap class could be elsewhere on the page (e.g. the 'Tabs' snippet). Also, IMO it seems too intrusive to override Bootstrap like this only for a snippet of the Website module.

After some testing, I noticed that simply checking if an instance of Scrollspy exists on `this.$scrollingElement` to do the "Dispose" seemed to solve the problem.

```js
if (this.$scrollingElement.data('bs.scrollspy')) {
    this.$scrollingElement.scrollspy('dispose');
}
```
Could you investigate this further?

Reply:
@bvr-odoo Thanks for the check! the idea was to prevent the update of `#top-menu` links, so we won't need to think about the issue when we trigger a `dispose` elsewhere... but as you mentioned, since this affects other snippets... the correct way is to fix it on the TOC. The check on `data('bs.scrollspy')` should work correctly :+1: since we make sure `$scrollingElement` has a `target` === `'.s_table_of_content_navbar'` which fixes the main cause of the issue (a missing target element...).